library/uri/generic/to_s_spec.rb

```diff
@@ -2,5 +2,9 @@
 require 'uri'
 
 describe "URI::Generic#to_s" do
-  it "needs to be reviewed for spec completeness"
+  ruby_version_is "3.2" do
+    it "returns proper url" do
```

Comment:
**minor:** a title is supposed to reflect checks in a test case so I would rephrase it somehow... Also it makes sense to add a comment with a link to the origin issue (for both new test cases).

```suggestion
    it "preserves / characters when host is empty" do
```